New get started and guides pages

ibek opened the pull request:
The get started and guides pages are linking to our new and latest community docs: https://docs.jboss.org/kogito/release/latest/html_single/

We were working on the changes together with @sterobin and @ricardozanini 

I also fixed the local dev configuration ... when I was running jekyll serve, images weren't loaded.

ibek committed:
Fixed jekyll dev environment

ibek committed:
New get started and guides pages

spolti commented on _site/404.html:
```diff
@@ -11,18 +11,18 @@
   <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.1.0/css/all.css" integrity="sha384-lKuwvrZot6UHsBSfcMvOkWwlCMgc0TaWr+30HWe3a4ltaBwTZhyTEggF5tJv8tbt" crossorigin="anonymous">
   <!-- Global site tag (gtag.js) - Google Analytics -->
-  <script async src="https://www.googletagmanager.com/gtag/js?id=UA-125236544-1"></script>
+  <script async src="https://www.googletagmanager.com/gtag/js?id=UA-157945096-1"></script>
```
is it needed to get statistics om google analytics?

ibek replied:
IDK, it automatically changed after I built the website. I agree there are much needed changes in the code itself but this PR is mostly about adding the new content to get started and guides pages. I didn't want to make too many changes at once, to the styling, css, fonts, layouts, etc.. I can remove the metrics completely if you want me to.

spolti replied:
I was just curios, @sterobin ^

spolti commented on _site/blog/welcome-to-jekyll/index.html:
```diff
@@ -70,19 +73,19 @@ <h1>Welcome to Jekyll!</h1>
   </a>
-  <a class="share-twitter" href="https://twitter.com/intent/tweet?text=&url=http://localhost:4000&via=project_twitter_username&related=project_twitter_username" rel="nofollow" target="_blank" title="Share on Twitter">
-    <img src="//assets/images/share-page/icons_social-twitter.png"/>
+  <a class="share-twitter" href="https://twitter.com/intent/tweet?text=&url=http://localhost:4000&via=kogito_kie&related=kogito_kie" rel="nofollow" target="_blank" title="Share on Twitter">
```
is this url correct?
localhost:4000

ibek replied:
This is generated content. I changed the twitter account to kogito_kie because I thought it might be used somewhere .. well, we should use twitter link in the web somewhere but I would add some icon / feed in some other PR.

cristianonicolai replied:
@ibek can this url value be changed to _https://kogito.kie.org/_. Otherwise, this localhost is shown in the tweet text.

ibek replied:
This is some welcome to jekyll page ... it's not used anywhere at https://kogito.kie.org AFAIK. I can remove the page completely or as I said I can add the share twitter button somewhere to the web but as part of another PR. This one alone focuses on adding the get started and guides content.

spolti commented on get-started.md:
```diff
@@ -1,5 +1,69 @@
+## Table of Contents
+### Getting Started
+
+ - [Kogito business automation overview](https://docs.jboss.org/kogito/release/latest/html_single/#con_kogito-automation)
```
shouldn't this be linked with the ## Kogito business automation overview topic as this topic already points to the same doc?

ibek replied:
No this is how it should behave as we agreed on with @sterobin  ... directly linking to the docs.

spolti commented on get-started.md:
```diff
@@ -1,5 +1,69 @@
+<div class="guide-item" markdown="1">
+## Creating and running your first Kogito services
+
+As a developer of business processes and rules, you can use Kogito business automation to build cloud-native applications that adapt to your business domain and tooling.
```
not sure if this phrase is ok here, at least for me the "applications that adapt to your business and tooling" is not clear.
, can you please clarify?

ibek replied:
Adding @sterobin to comment

sterobin replied:
@spolti , Ivo is just copying from the approved doc content. That messaging was just what SMEs and QE agreed on so far. Refers to the fact that Kogito is designed to be implemented as part of your existing development workflow, tool set (VSCode), and business domain, instead of forcing you to change the tools and processes you use to adapt to Kogito requirements. This is intentionally only brief. It's expanded in the Kogito overview in the doc.

Feel free to comment with specific suggestions in this PR that's currently under final QE verification: https://github.com/kiegroup/kie-docs/pull/2308

spolti replied:
IMHO we could improve it, basically you have only two available tooling (maybe 3 with the chrome plugin), bpmn|dmn.new or the vscode plugin, if a developer uses eclipse or intelliJ he will have to adapt their development tools to design kogito services, does it make sense?

sterobin replied:
@spolti , yep, makes sense. Though apparently others disagree since that was the sanctioned messaging, haha! I saw your feedback on that other PR. Thank you! Feel free to bring this up as well.

spolti commented on get-started.md:
```diff
@@ -1,5 +1,69 @@
+* <a href="https://docs.jboss.org/kogito/release/latest/html_single/#proc_kogito-testing-app">Testing a Kogito service</a>
+* <a href="https://docs.jboss.org/kogito/release/latest/html_single/#ref_kogito-app-examples_kogito-creating-running">Example applications with Kogito services</a>
+
+<a href="https://docs.jboss.org/kogito/release/latest/html_single/#chap_kogito-creating-running" class="button-cta secondary">START CODING</a>
```
Is this button correct? I mean, it points to already seen topics like Enabling kogito vscode extension.

ibek replied:
Yes it points to the whole section "Creating and running your first Kogito services" where all the topics above are covered.

spolti replied:
alright.

spolti commented on guides.md:
```diff
@@ -0,0 +1,125 @@
+<div class="guide-item" markdown="1">
+### Deploying Kogito services on OpenShift using the OpenShift web console
+
+After you create your Kogito services as part of a business application, you can use the OpenShift web console to deploy your services.
```
after creating your..?

ibek replied:
This is a copy&paste from the docs. If we change it, we should change it in docs as well. Adding @sterobin  to advise.

sterobin replied:
@spolti , just less clear, in terms of "global English style" conventions. Plus would introduce an inconsistency with the rest, in the docs anyway.

spolti reviewed (changes requested):
just a minor comments, nice content.

ricardozanini commented on _includes/get-started-band.html:
```diff
@@ -0,0 +1,31 @@
+      <div class="step">2</div>
+      <div class="step-desc">
+        <div class="step-desc">
+          You need a <a href="https://adoptopenjdk.net/" target="_blank">JDK 8 or JDK 11</a> with JAVA_HOME system property set.
```
Only JDK 11 will be supported

ibek replied:
Ok, I will change that.

ricardozanini reviewed (approved):
One tiny little thing. Many thanks for reviewing the Web Site, @ibek!

evacchi reviewed (approved):
loving this. Only (partially related) note to @sterobin, if possible I would much prefer to provide the HTML split version and link to that. We can include both versions, single and split, so people can ctrl-f single if they wish, and we can link to single sections (like these tutorials). Otherwise, as somewhere already noted, loading will jump up and down making it confusing for first-time readers.

ibek commented:
Is there anything else needed to be done or can someone merge it?

sterobin commented:
@evacchi , sure, we can do whatever we want for GA. For Dev Preview, we just needed an MVP to get the content out there. If someone wants to volunteer to create a split-up version for Dev Preview, anyone is welcome.

ricardozanini commented:
I can merge this. Are we ready @ibek @sterobin ?

ibek commented:
I updated the links and added new content and I believe we can merge it now. @krisv @sterobin @ricardozanini 

Also because we have the release notes I put a link to the homepage with the latest version acquired automatically from the latest released tag in the kogito-runtimes repo. See below:

![Screenshot from 2020-04-16 12-18-14](https://user-images.githubusercontent.com/1140579/79445626-42985180-7fdd-11ea-892e-ff1e8030da73.png)
![Screenshot from 2020-04-16 12-18-32](https://user-images.githubusercontent.com/1140579/79445628-4330e800-7fdd-11ea-84a0-2a016566d6ce.png)
![Screenshot from 2020-04-16 12-18-41](https://user-images.githubusercontent.com/1140579/79445630-43c97e80-7fdd-11ea-9906-8c31512f3b38.png)

sterobin commented:
@ibek , looks great! One thing: Can we remove "Using the Kogito Operator (coming soon)"? The exact nature of that content is still TBD, and I'd rather not make any promises. It might be merely reference content or even just dumped into community from the README.

ibek commented:
@sterobin ok I removed that item

ibek commented:
@krisv @sterobin @ricardozanini  can we merge it now?

sterobin commented:
@ibek , good with me. Thanks.

ricardozanini commented:
LGTM

ricardozanini commented:
@spolti looks good to you?

krisv commented:
Looks great, but let's time the merge as we need 0.9.1 released before these changes should land
